common/helpers/move/get-tag-lists.js

```diff
@@ -0,0 +1,21 @@
+const presenters = require('../../presenters')
+
+const getMoveUrl = require('./get-move-url')
+
+function getTagLists(move) {
```

Comment:
If this was accidentally called with an undefined move, what should happen?

Reply:
Good question